app/templates/image-set/cv-results.hbs

```diff
@@ -0,0 +1,44 @@
+<h3>CV Results</h3>
+
+<div class='row'>
+  <div class='col-sm-4'>
+    <ul class='list-group'>
+      {{#each model as |cvResult|}}
+        {{lg-cv-result-summary cvResult=cvResult action='setActiveCvResult'}}
```

Comment:
each result is a single image matching with a probability. So a single CV request will cause multiple CV Result objects to be created in the DB.